src/components/TooltipButton/TooltipButton.scss

```diff
@@ -0,0 +1,209 @@
+@import "constants/style.scss";
+
+$tooltip-button-size: 42px;
+
+.tooltip-button {
+  height: $tooltip-button-size;
+  width: $tooltip-button-size;
+  border-radius: calc($tooltip-button-size / 2);
+  background-color: transparent;
+  border: none;
+  padding: 0;
+  position: relative;
+  cursor: pointer;
+  pointer-events: auto;
+  box-sizing: border-box;
+
+  &:focus-visible {
+    outline: 2px solid #00000050;
+  }
+}
+
+.tooltip-button--active {
+  &:hover,
+  &:enabled:active {
+    .tooltip-button__icon {
+      &:nth-of-type(1) {
+        visibility: hidden;
+      }
+      &:nth-of-type(2) {
+        visibility: visible;
+      }
+    }
+  }
+
+  .tooltip-button__icon {
+    background-color: $menu-icon-background-color--dark;
+    color: $color-white;
+  }
+}
+
+.tooltip-button__tooltip {
+  position: absolute;
+  top: -58px;
+  left: 18px;
```

Comment:
![image](https://user-images.githubusercontent.com/44020029/185057855-5883953a-8a30-4a71-80db-cc85cb7a9a1e.png)

But I cant think of a better way, so leave it

Reply:
Thanks for spotting that, we can actually completely remove it. The position is overwritten in a media query for larger screens, and on smaller screens we don't display the tooltips at all.